index.html

```diff
@@ -0,0 +1,107 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="stylesheets/mainpagestyle.css">
+  <title>Film Festival Chennai 2022</title>
+</head>
+
+<body>
+  <header id="control-nav-bars">
+    <a href="#" class="hamburger-wrapper"><img class="hamburger-icon" src="bg-icons/hamburger-icon.svg" alt="hamburger menu icon"></a>
+    <div class="control-bar">
+      <ul class="control-list no-style">
+        <li class="control-list-item"><a href="https://www.facebook.com" target="_blank" rel="noopener"><img class="social-logo" src="bg-icons/facebook-icon.svg" alt="Facebook icon"></a></li>
+        <li class="control-list-item"><a href="https://www.twitter.com" target="_blank" rel="noopener"><img class="social-logo" src="bg-icons/twitter-icon.svg" alt="Twitter icon"></a></li>
+        <li class="control-list-item"><a href="">English</a></li>
+        <li class="control-list-item"><a href="">MyPage</a></li>
+        <li class="control-list-item"><a href="">Logout</a></li>
+      </ul>
+    </div>
+    <div class="navigation-bar">
+      <a href="index.html"><img class="logo-image" src="logo-resources/navbar-logo.svg" alt="image of video camera and title"></a>
+      <ul class="navigation-list no-style">
+        <li class="navigation-list-item"><a class="orange-text" href="about.html">About</a></li>
+        <li class="navigation-list-item"><a href="">Schedule</a></li>
+        <li class="navigation-list-item"><a href="">Venues</a></li>
+        <li class="navigation-list-item"><a href="">Partner</a></li>
+        <li class="navigation-list-item"><a href="">News</a></li>
+        <li class="navigation-list-item orange-box"><a href="">Tickets</a></li>
+      </ul>
+    </div>
+  </header>
+  <section class="section-intro">
+    <small class="header-welcome">"Hello! Vanakkam"</small>
+    <h1 class="header-title">TAMIL FILM ASSOC.<br>FILM FESTIVAL<br>2022</h1>
+    <p class="intro-description">Most Popular Film Festival in South India, gathering film talents from all
+      over the World. More than
+      20 films are to be screened throughout the week-long festival. And much more. </p>
```

Comment:
- [ ] I would recommend that you reduce the width of this section and also adjust the `font-size` to match the original design
 
![image](https://user-images.githubusercontent.com/98453979/210374635-621bffcf-69e3-4318-8381-6bd13b5c77ce.png)

### Checkout the original design

![image](https://user-images.githubusercontent.com/98453979/210374842-944426ff-042e-4d27-b92f-720506596a13.png)